tests/net_inet/asyncio_http_https_opc.py

```diff
@@ -0,0 +1,81 @@
+# Test simple HTTP request with asyncio.open_connection()
+
+try:
+    import os
+    import asyncio
+    import ssl
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+
+async def http_get(url):
+    reader, writer = await asyncio.open_connection(url, 80)
+
+    print("write GET")
+    writer.write(b"GET / HTTP/1.0\r\n\r\n")
+    await writer.drain()
+
+    print("read response")
+    data = await reader.read(100)
+    print("read:", data.split(b"\r\n")[0])
+
+    print("close")
+    writer.close()
+    await writer.wait_closed()
+    print("done")
+
+
+# Test simple HTTPS request with uasyncio.open_connection()
+
+
+# This certificate was obtained from micropython.org using openssl:
+# $ openssl s_client -showcerts -connect micropython.org:443 </dev/null 2>/dev/null
+# The certificate is from Let's Encrypt:
+# 1 s:/C=US/O=Let's Encrypt/CN=R3
+#   i:/C=US/O=Internet Security Research Group/CN=ISRG Root X1
+# Validity
+#            Not Before: Sep  4 00:00:00 2020 GMT
+#            Not After : Sep 15 16:00:00 2025 GMT
+# Copy PEM content to a file (certmpy.pem) and convert to DER e.g.
+# $ openssl x509 -in certmpy.pem -out certmpy.der -outform DER
+# Then convert to hex format, eg using binascii.hexlify(data).
+
+ca_cert_chain = "mpycert.der"
+
+try:
+    os.stat(ca_cert_chain)
+except OSError:
+    print("SKIP")
+    raise SystemExit
+
+client_ctx = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+client_ctx.verify_mode = ssl.CERT_REQUIRED
+client_ctx.load_verify_locations(cafile=ca_cert_chain)
+if hasattr(client_ctx, "load_default_certs"):
+    client_ctx.load_default_certs()
+
+
+async def https_get(url, port, sslctx):
+    reader, writer = await asyncio.open_connection(url, port, ssl=sslctx)
+
+    print("write GET")
+    writer.write(b"GET / HTTP/1.0\r\n\r\n")
+    await writer.drain()
+
+    print("read response")
+    data = await reader.read(100)
+    print("read:", data.split(b"\r\n")[0])
+
+    print("close")
+    writer.close()
+    await writer.wait_closed()
+    print("done")
+
+
+async def http_https_get(url1, url2, port, sslctx):
+    await http_get(url1)
+    await https_get(url2, port, client_ctx)
```

Comment:
Is it necessary to test non-SSL followed by SSL?  Non-SSL connections are already tested by existing tests, so it'd be good if this test just did the SSL test, ie just called `https_get()`.

Then the `asyncio_https_http_opc.py` test is not needed.